pkg/cmd/init/options.go

```diff
@@ -32,31 +33,30 @@ func NewInitOptions() *InitOptions {
 
 func (o *InitOptions) Complete(args []string) error {
 	if o.Online { // use online templates, official link or user-specified link
-		o.TemplateNameOrURL = getURL(args)
+		o.TemplateRepoPath = templateRepoPathFromURL(args)
 	} else { // use offline templates, internal templates or user-specified local dir
-		path, err := getPath(args)
+		path, err := templateRepoPathFromLocal(args)
 		if err != nil {
 			return err
 		}
-		o.TemplateNameOrURL = path
+		o.TemplateRepoPath = path
 	}
 	return nil
 }
 
 func (o *InitOptions) Validate() error {
-	if o.Online {
-		return nil
-	}
-	// offline mode may need to generate templates
-	if err := validatePath(o.TemplateNameOrURL); err != nil {
-		return err
+	if !o.Online {
```

Comment:
nesting `if` statements isn't a cool code style

Reply:
actually, I think it's okay, maybe you can give me an example :)

Reply:
original style is an example, it's called "guard clauses"

Reply:
haha, the original code is also provided by me. I changed it because maybe there will be validation when online is true. Your suggestion is better for now, I'll change it.